demo/index.html

```diff
@@ -171,6 +171,35 @@ <h2 id="nested-heading">nested</h2>
     </div>
   </div>
 
+  <h2 id="sibling-heading">sibling traps</h2>
+  <p>
+    Sibling focus traps. When the second trap is deactivated, focus returns to the first trap.
+  </p>
+  <div id="sibling-first" class="trap" tabindex="-1">
+    <p>
+      <button id="activate-first-sibling">
+        activate first trap
+      </button>
+    </p>
+    <p>
+      <button id="deactivate-first-sibling">
+        deactivate first trap
+      </button>
+    </p>
+    <p>
+      <button id="activate-second-sibling">
+        activate second trap
+      </button>
+    </p>
+  </div>
+  <div id="sibling-second" class="trap" tabindex="-1" style="display: none;">
+    <p>
+      <button id="deactivate-second-sibling">
```

Comment:
When I click this button, should the second trap area disappear? I find it a little confusing that it appears when I activate the trap but does not disappear when I close it.

Reply:
That's very odd, it does disappear for me -

![disappearing box](https://d.pr/i/eML2yS.gif)

Reply:
@chandlerprall I think it will disappear if you click the deactivate button but not if you hit Escape.

Reply:
Aha, thanks. I've updated the example to reset the divs on deactivation event.